### DISCUSSION TITLE
Overriding parent team repository access

### DISCUSSION THREAD

**Discussion by ewpratten** [community/community#23660] [New to GitHub]
<p>I have an organization where all users have access to everything, and some teams in it. If I “invite” a team to an org repo, and set that team’s access to “read only”. Does this override the default write access?</p>
<p>I basically need to block a team from writing to a repository.</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi <a class="mention" href="/ewpratten">@Ewpratten</a> :wave:</p>
<p>The team permissions will take precedent. However, if an individual user has a role higher than Member, they will still have write access.</p>
